go.mod

```diff
@@ -30,6 +30,7 @@ require (
 	github.com/mattn/go-isatty v0.0.20 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	github.com/rogpeppe/go-internal v1.12.0 // indirect
+	github.com/substrait-io/substrait-go v1.2.0 // indirect
```

Comment:
Something is pointing to the old version of substrait-go, it should point to v3.